.ebextensions/nginx-upload-size.config

```diff
@@ -2,3 +2,8 @@ files:
   /etc/nginx/conf.d/proxy.conf:
     content: |
       client_max_body_size 50M;
+
+
+container_commands:
+  01_reload_nginx:
+    command: "service nginx reload"
```

Comment:
@AlexanderUlitin 
The file proxy.conf is copied from the unzipped deployment only after nginx is restarted. so after each deployment, we are restarting nginx manually using Container command, once we find a solution to only run once whenever a new instance is created we will modify this file.